### DISCUSSION TITLE
auto placement with initial placement

### DISCUSSION THREAD

**Discussion by fictivecreations** [floating-ui/floating-ui#1227] [Q&A]
Is there anyway in the options popper.js has to have a popover that's set to say 'bottom-start' as the initial placement value, and have it adjust to it's available space if things change, by going into any other placement, _without_ having to specify the flip modifier options with every single available placement as a fallback?

I feel like that should be a pretty common use case with an absolutely abysmal way to get to it, but I want to make sure I'm not missing something.

**Comment by FezVrasta** [comment]
Have you tried to set "auto" as fallback placement?

**Reply by fictivecreations** [reply]
I had not! And that's definitely _much_ closer to what I was hoping for. 

I kind of wish it was a little smarter - as in when I start with 'right-start' it would flip with a preference for a '-start' placement. I recognize that I can do this with 'auto-start' as a fallback but as I'm setting up a more dynamic component, so I'd have to do the string matching myself to make sure I actually match what's sent in. But that's definitely a much more complex ask and this at least gets me on a better path. Thank you!
